spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -193,6 +193,13 @@
             <optional>true</optional>
         </dependency>
 
+        <!-- Security -->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-oauth2-resource-server</artifactId>
```

Comment:
It's odd to see a starter imported here. This module is just a pom. Can we be more specific about which modules we need here? Is it `spring-security-oauth2-resource-server` and something else?

Reply:
Ah! Right. We must not use starters in the `autocofigure` module as dependency.
I don't remember "Why?", but that is what @snicoll recommended us to do.